src/components/styles/ModalDesign.js

```diff
@@ -25,7 +24,38 @@ export const PlayerStyle = styled.div`
 export const IFrameStyler = styled.iframe`
   top: 0;
   left: 0;
-  width: 100%;
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+  width: 80%;
   height: 100%;
   position: absolute;
+  text-align: center;
+`;
+
+export const DescriptionField = styled.span`
+  padding: 10px;
```

Comment:
You could use the shorthand here 
```suggestion
  padding: 10px 10px 20px 10px;
```